### ISSUE TITLE
Cannot delete photos!

### ISSUE THREAD

**Issue by CloudyPadmal** [fossasia/phimpme-android#107]
**Actual Behaviour**

The photos in the gallery, cannot be deleted

**Expected Behaviour**

If the user wants to delete an image or a set of images of his selection,  he must be provided with an option to delete.

**Steps to reproduce it**

1. Open app and go to gallery
2. Try deleting an image

**LogCat for the issue**

> Not available

**Screenshots of the issue**

> Not available

**Would you like to work on the issue?**

If this feature is okay to implement, then I'll be happy to work on this!

**Comment by Abhi2424shek** [comment]
Ok pls work on.

**Comment by CloudyPadmal** [comment]
Oki dokie :D

**Comment by CloudyPadmal** [comment]
@Abhi2424shek there's a little issue! In some lines there are tabs and some lines spaces! This causes indentation mismatches in GitHub. Should I make them all tab indented? :thinking: 

![Pic 1](https://cloud.githubusercontent.com/assets/14261304/23822599/5c955310-0675-11e7-85e8-c4dabf26d64b.png)
![Pic 2](https://cloud.githubusercontent.com/assets/14261304/23822605/81b5ca08-0675-11e7-9f13-5e5e7f58f53d.png)

**Comment by Abhi2424shek** [comment]
Open up another issue, we should use whitespaces instead of tabs IMO. @pa1pal what do you think? Spaces are uniform on all editors!

**Comment by CloudyPadmal** [comment]
@Abhi2424shek I guess most of the files have mixed up tabs and spaces. I choose one at random;

![random](https://cloud.githubusercontent.com/assets/14261304/23823922/b7374e78-0692-11e7-9755-4b372c60cea2.png)

When changing it to a consistent format, it will cause a lot of conflicts with the PRs others have made and to be made right? :(

I'll open an issue? :D

**Comment by Abhi2424shek** [comment]
We will resolve it when the PRs are all dealt with it, we will solve the issue only after there are no PRs! That would take time!

**Comment by CloudyPadmal** [comment]
@Abhi2424shek great idea! :bulb: :100:

**Comment by optnio00** [comment]
This problem is solved. Now user can select a image from Home Tab and and after that user choose option delete to delete that image
![screenshot_2017-03-16-23-40-38](https://cloud.githubusercontent.com/assets/15853647/24012616/9aa5ab90-0aa4-11e7-9c5a-47ff13dbd23f.png)
